browser/src/Services/EditorManager.ts

```diff
@@ -93,9 +93,22 @@ class AllEditors implements Oni.Editor {
         return this._activeEditor.neovim
     }
 
-    public openFile(file: string): Promise<Oni.Buffer> {
-        Log.warn("Not implemented")
-        return Promise.resolve(null)
+    public async openFile(file: string, method = "edit"): Promise<Oni.Buffer> {
+        const cmd = new Proxy(
```

Comment:
Could we just have this call into the `activeEditor`'s `openFile` method? I'd like for the `EditorManager` to be agnostic to `Neovim`, if possible - ideally, the `EditorManager` could support all sorts of `IEditor` implementations - even ones that didn't depend on Neovim. Looks like this is duplicated with `NeovimEditor`'s implementation anyway.